### PR TITLE
Backup: removed backup-in-progress animation on small viewports

### PR DESCRIPTION
**Pull request by monsieur-z**
Fixes 1164141197617539-as-1201305007019351

#### Changes proposed in this Pull Request:
In the Backup plugin, the backup-in-progress animation hides the progress bar and the text around. This PR fixes this by hiding the animation on viewports narrower than 960px.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

- Download this PR and build the Backup plugin
- Launch your local testing site and makes sure it has the local version of the Backup plugin installed
- Get a subscription to Backup
- Visit the `Jetpack > Backup` section in WPadmin
- You should see the backup-in-progress animation once the first backup starts (*)
- Check that the animation is hidden for viewports narrower than 960px

_* If your site already has previous backups, you may not be able to visualize this screen. In that instance, force the call of `renderInProgressBackup` at the bottom of `Backup.js`, and rebuild the plugin_.

__Expected behaviour__

https://user-images.githubusercontent.com/1620183/139920545-d27d330e-d4ba-41f0-846d-666c72293036.mov

### CODE REVIEW

**Review by jwebbdev — DISMISSED** [review]
Works as described 👍

**Review by sdixon194 — APPROVED** [review]

**Comment by monsieur-z** [comment]
Thanks for the _Status_ update, @sdixon194. I've rebased the branched, as required by the checks list.

Can you refresh my memory: is there something else I need to do on my side? Or will the branch be merged into master as part of the plugin release process?

**Review by sdixon194 — APPROVED** [review]

**Comment by sdixon194** [comment]
>  I've rebased the branched, as required by the checks list.

Thanks for rebasing! If you do a master merge instead of rebasing, it will retain the approval as well. 

Once it's approved and ready to merge, you can push the merge button if you want to bring it to master :) It will be released in the next version of Jetpack, 10.4, early next month.

**Pull request merged**
